src/sonic-config-engine/minigraph.py

```diff
@@ -1458,6 +1458,29 @@ def select_mmu_profiles(profile, platform, hwsku):
                 base_file = os.path.join(path, file_item)
                 exec_cmd(["sudo", "cp", file_in_dir, base_file])
 
+def is_ipv4(address):
+    # address maybe is bytes type, so encode and decode to unicode
```

Comment:
>address maybe is bytes type [](http://example.com/codeflow?start=6&length=27)

Thanks for the comment, but I did not get the reason. Why the address could be bytes? Will it just throw if it is bytes? Is it throwing expected?

Reply:
The throw is not expected, when render config template the sonic-cfggen will throw will following exception:

2024-01-04T06:58:02.4821234Z ipaddress.AddressValueError: 'FC00:2::32/64' does not appear to be an IPv4 or IPv6 network. Did you pass in a bytes (str in Python 2) instead of a unicode object?

Reply:
I got your point. Could you check whether `UNICODE_TYPE` is a solution, which is already used in this file. The benefit is better performance for both python2 and 3.

Reply:
Fixed